database/bigMenuData.js

```diff
@@ -2,67 +2,68 @@ const faker = require('faker');
 const fs = require('file-system');
 const path = require('path');
 const csvWriter = require('csv-write-stream');
-const headers =  {
-  headers: [
-    'id',
-    'mealType',
-    'Item',
-    'Des',
-    'Price'
-  ]
+const headers = {
+  headers: ['Breakfast', 'Lunch', 'Dinner', 'Brunch', 'HappyHour']
 }
 const writer = csvWriter(headers);
 
+  var generateMenuObject_100000_helper = function() {
```

Comment:
I appreciate how you handle the "realistic data" criteria here with randomize array lengths